modules/cli/src/main/scala/scala/cli/commands/SharedCompilationServerOptions.scala

```diff
@@ -185,26 +185,28 @@ final case class SharedCompilationServerOptions(
         BloopVersion(Constants.bloopVersion)
       ))(v => BloopRifleConfig.Strict(BloopVersion(v)))
 
-  def bloopDefaultJvmOptions(logger: Logger): List[String] = {
-    val filePath = os.Path(bloopGlobalOptionsFile, Os.pwd)
-    if (os.exists(filePath) && os.isFile(filePath))
-      try {
-        val json = ujson.read(
-          os.read(filePath: os.ReadablePath, charSet = Codec(Charset.defaultCharset()))
-        )
-        val bloopJson = upickle.default.read(json)(BloopJson.jsonCodec)
-        bloopJson.javaOptions
-      }
-      catch {
-        case e: Throwable =>
-          System.err.println(s"Error parsing global bloop config in '$filePath':")
-          e.printStackTrace()
+  def bloopDefaultJvmOptions(): Option[List[String]] = {
+    val filePathOpt = bloopGlobalOptionsFile.map(os.Path(_, Os.pwd))
```

Comment:
NIT (allows to ignore `--bloop-global-options-file ""`)
```suggestion
    val filePathOpt = bloopGlobalOptionsFile.filter(_.trim.nonEmpty).map(os.Path(_, Os.pwd))
```